'Footer_Form_Microanimation'

shreyanshu07 opened the pull request:
Here I have added Microanimation from the below link
https://codepen.io/MichaelArestad/pen/ohLIa
I have matched the color theme with the website and checked the reponsiveness was same .
![image_2021-06-28_204535](https://user-images.githubusercontent.com/72155853/123661025-cee50c00-d851-11eb-9ee8-e35cadf15779.png)
![image_2021-06-28_204629](https://user-images.githubusercontent.com/72155853/123661123-eb814400-d851-11eb-8923-16e9153ff3e4.png)

smaranjitghose commented:
## Feedback:
- Please have a look at the entire micro animation.
- The color of the labels i.e. Name, Email, and Message (this missing) should be the same as that of the Send Button.
- Unresponsive for certain mobile screens:

![Mobile_Footer](https://user-images.githubusercontent.com/46641503/123684501-293e9680-d86b-11eb-8d97-af6055b98c33.png)

smaranjitghose commented on assets/css/Check.css:
```diff
@@ -0,0 +1,115 @@
+@import "compass/css3";
+ @import url(https://fonts.googleapis.com/css?family=Open+Sans:400,700,600,300,800);
+
```
What is this file for?

smaranjitghose commented on assets/css/Check.css:
```diff
@@ -0,0 +1,115 @@
+	 transform: translateX(100%);
+     
+}
+#yourName {
```
Name the variables judiciously!

smaranjitghose commented on assets/js/app.js:
```diff
@@ -135,10 +135,19 @@ let footer = $(`
               </div>
                 <form>
```
The above files are renamed without any requirement leading to error in other sections

smaranjitghose commented on assets/js/app.js:
```diff
@@ -135,10 +135,19 @@ let footer = $(`
-                  <input type="email" name="field2" placeholder="Email Address"  required/>
+                    <link rel="stylesheet"  href="assets/css/Check.css">
+                  <div class="row1">
+  <span>
```
Use a Linter please. This is poorly written

smaranjitghose commented on assets/js/app.js:
```diff
@@ -135,10 +135,19 @@ let footer = $(`
+    <input class="skinnyx" id="yourName" type="text" placeholder="Enter your Name" /><label for="Name">Name</label>
+  </span>
+  <span>
+    <input class="skinnyx" id="yourEmail" type="email" placeholder="Enter your Email-id" /><label for="EmailId">Email</label>
```
Name the classes judiciously!

shreyanshu07 commented:
I have made the following changes :-
1. Changed the names of variables , classes
2. Gave proper indentation in the code
3. Removed the unwanted header files in my check.css file
4. Changed the color of label according to the requirements.
5. Added the same to the "Message" field

shreyanshu07 commented:
@deeqakkk 
![image](https://user-images.githubusercontent.com/72155853/123823509-e805bf80-d91a-11eb-9318-2fe02717fb44.png)

shreyanshu07 commented:
![image](https://user-images.githubusercontent.com/72155853/123823942-3e72fe00-d91b-11eb-8e24-0a8225466ea3.png)
@deeqakkk I think now it's perfect :-)

smaranjitghose commented:
Still not mobile responsive

shreyanshu07 commented:
![image](https://user-images.githubusercontent.com/72155853/123846269-c6183700-d932-11eb-8804-6f29a62d6beb.png)
@smaranjitghose
